tests/smoke/features/shareFileJourney.feature

```diff
@@ -7,38 +7,50 @@ Background:
     | Alice |
     | Brian |
   And admin set the default folder for received shares to "Shares"
+  And admin disables auto accepting
 
 Scenario: Alice shares folder with file to Brian
   Given "Alice" has logged in
-  Then "Alice" opens the "files" app
-  Then "Alice" navigates to the files page
-  Then "Alice" creates following folders
-    | folder_to_shared           |
-    | folder_to_share1/subFolder |
+  When "Alice" opens the "files" app
+  And "Alice" navigates to the files page
+  And "Alice" creates following folders
+    | folder_to_shared |
   And "Alice" uploads following resources
     | resource  | to               |
-    | lorem.txt | folder_to_shared |             
-  Then "Alice" moves following resources
-    | resource                             | to                         |
-    | folder_to_shared/lorem.txt           | folder_to_share1/subFolder |
-    | folder_to_share1/subFolder/lorem.txt | All files                  |
-    | lorem.txt                            | folder_to_shared           |
-  Then "Alice" copies following resources
-    | resource                             | to                         |
-    | folder_to_shared/lorem.txt           | folder_to_share1/subFolder |
-    | folder_to_share1/subFolder/lorem.txt | All files                  |
```

Comment:
Why remove those steps? I suppose @fschade went to test a couple of different scenarios here (Copying to home / folder / subfolder)